uw-research-computing/os-transition-htc.md

```diff
@@ -1,16 +1,14 @@
 ---
 highlighter: none
 layout: markdown-page
-title: HTC System Transitioning to a New Linux Version (CentOS Stream 8)
+title: HTC System Transition to a New Linux Version (CentOS Stream 8)
 published: true
 ---
 
 Starting on August 1, 2022, CHTC's high throughput computing (HTC) system began upgrading
-the Linux distribution and version we use on our servers. The current version of Linux running on 
-most of our servers is CentOS 7 and we will be upgrading to CentOS Stream 8. **This 
-transition will happen over the next several weeks ([See "Important Dates"](#important-dates))
-and requires [proactive action from CHTC users](#what-you-need-to-do) to prevent 
-interruptions to research work.** 
+the Linux distribution and version we use on our servers. CentOS Stream 8 is now 
+the operating system used on the majority of our servers, and the default requested 
+operating system by jobs, unless specified otherwise. 
```

Comment:
```suggestion
the Linux distribution and version we use on our servers. **CentOS Stream 8 is now 
the operating system used on the majority of our servers, and the default requested 
operating system by jobs, unless specified otherwise.** 
```